Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN --mount=type=cache,target=/go/pkg/mod \
     --mount=type=cache,target=/root/.cache/go-build \
     go build -trimpath -ldflags="-s -w" -o bin/service
 
-FROM --platform=$BUILDPLATFORM node:17.7-alpine3.14 AS client-builder
+FROM node:17.7-alpine3.14@sha256:539e64749f7dc6c578d744d879fd0ec37f3afe552ae4aca9744cc85121728c4c node:17.7-alpine3.14 AS client-builder
```

Comment:
@cdupuis @gtardif It seems the `--platform=$BUILDPLATFORM` is mistakenly replaced instead of the actual image

Reply:
Oh, that's a bug. Fixing....